### PR TITLE
Python 3 support and compatibility with Ghost 2 import format

### PR DESCRIPTION
**Pull request by joelw**
Hi! Thanks very much for writing this script. I'm not sure if you're still maintaining it, but I have made some changes to:

* Make the script run under Python 3
* Support the new Ghost 2 JSON import format 
* Ensure that slugs are unique, avoiding import collisions

### CODE REVIEW

**Comment by jpadilla** [comment]
Oh hey, this is nice. I'm currently not using ghost or tumblr either so can't test this out, but would gladly merge it.  Any reason why pandoc is no longer needed?

**Comment by joelw** [comment]
There was a Python 3 incompatibility with the version of pandoc in requirements.txt, and it wasn't clear to me that pandoc was still being used - the code was doing:

doc = pandoc.Document()
doc.html = unidecode(body)

But then doc isn't referenced again, and the exported object just uses 'body' directly. It seems to work just fine without it! Do you remember what it was needed for originally?

File: requirements.txt [hunk 1]
```diff
@@ -7,4 +7,3 @@ gunicorn==18.0
 itsdangerous==0.23
 -e git://github.com/kennethreitz/pyandoc.git@04cd95d3426c7665fc88a1774709e7b68fe226d7#egg=pyandoc-master
```
Comment (jpadilla) [inline requirements.txt]: If we're not using pandoc, let's remove this requirement.

Comment (joelw) [reply requirements.txt]: Oops, sorry for the oversight! I've updated the PR.

**Comment by jpadilla** [comment]
Thanks! I'll try to get this deployed soon.

**Comment by jpadilla** [comment]
Just deployed this, thanks again.

**Comment by joelw** [comment]
It's been a pleasure! :)

**Pull request merged**